scripts/conflicts/diff.js

```diff
@@ -22,35 +28,94 @@ console.log(`Generating diffs for conflicts on ${server}`);
 
 const DB = new PouchDB(server);
 
-DB.query('medic-conflicts/conflicts')
+const writeToFile = function(path, content) {
+  content  = JSON.stringify(content, null, 3); 
+  fs.writeFile(path, content, (err) => {
+    if(err){
+      console.log(err);
+    }
+  });
+};
+
+const createDirectory = function(conflictDirectoryPath) {
+  if (!fs.existsSync(conflictDirectoryPath)){
+    try {
+      fs.mkdirSync(conflictDirectoryPath);
+    } catch(err){
+      console.log(err);
+    }
+  }
+};
+
+const isDiectoryEmpty = function isEmpty(path) {
+  return fs.readdirSync(path).length === 0;
+};
+
+const getConflictDirectoryPath = function (r, mainConflictFilesDirectory) {
+  let conflictDirectoryName = r.id;
+  if (r.id.includes('target')) {
+    conflictDirectoryName = r.id.split(':')[0];
```

Comment:
Please add a comment that explains what is happening here. 
The comment should contain: 
1. An example document _id that matches `target`
2. The expected directory name based on that example